test-add/test.js

```diff
@@ -0,0 +1,14 @@
+const add = require('./add.js');
+
+const test = (a, b, c) => {
+  if (add(a, b) === c) {
+    return 'Correct';
+  }
+  return 'Incorrect';
+};
+
+console.log(test(1, 2, 3));
```

Comment:
wouldn't it be more efficient to move console.log into the function so you don't have to repeat it over and over?